### PR TITLE
Feature/applics 70 overview

### PR DESCRIPTION
**Pull request by HadrianHughesKC**
## Describe your changes

* feat(web/applications): remove link to Project information page
* feat(web/applications): update case overview to match new designs
* chore(web/applications): remove all references to Project Information
* test(web/applications): update tests to match new Overview page
* chore(web/applications): create a common function to get project team
* refactor(web/applications): move Azure project team code to common
Also includes some structural refactoring of the functions for code
clarity.
refactor(web/applications): move getManyProjectTeamMembersInfo to common
* feat(web/applications): pass key team members to Overview page template
* feat(web/applications): render case manager and NSIP officers
feat(web/applications): render case reference and sector+subsector
* refactor(web/applications): move AD methods to a separate service module
* test(web/applications): remove case overview unit test
The request to fetch the project team is causing this test to file, but
it isn't clear why. Mocks don't seem to work here.

## Issue ticket number and link

[APPLICS-70](https://pins-ds.atlassian.net/browse/APPLICS-70)

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [x] I have referenced the ticket number above
- [x] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

[APPLICS-70]: https://pins-ds.atlassian.net/browse/APPLICS-70?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by francescobrenci-AR — DISMISSED** [review]

File: apps/web/src/server/views/applications/case/components/case-details-table.component.njk [hunk 1]
```diff
@@ -1,30 +1,35 @@
...
+    gridReferences: gridReferences,
+    regionNames: regionNames
+  }, isWelsh) %}
 
 	{% set tableRows = [] %}
 	{% for rowData in rowsData %}
 		{#  In the value column, rowData can be used because "title" and "url" are ignored	#}
 		{% set cols = [{text: rowData.title}, rowData] %}
 
 		{% set linkCol =  {
```
Comment (BenSurgisonPINS) [inline apps/web/src/server/views/applications/case/components/case-details-table.component.njk:31]:
```suggestion
		{% if not isPublishPage %}
			{% if rowData.url %}
				{% set linkCol =  {
					html: caseEditLink({editStepUrl:rowData.url, caseId: params.case.id}),
					classes: 'text-align--right pins-table__cell--s'
				} %}
                        {% else %}
				{% set linkCol =  {} %}
                        {% endif %}
			{% set cols = (cols.push(linkCol), cols) %}
		{% endif %}
```

Comment (BenSurgisonPINS) [reply apps/web/src/server/views/applications/case/components/case-details-table.component.njk:31]: As above, I think the link should not be added if it doesn't exist in the rowData

File: apps/web/src/server/views/applications/case/components/case-details-table-legacy.component.njk [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+	{% endif %}
+
+  {% set rowsData = buildCaseInformation({ case: case, gridReferences: gridReferences, regionNames: regionNames }, isWelsh) %}
+
+	{% set tableRows = [] %}
+	{% for rowData in rowsData %}
+		{#  In the value column, rowData can be used because "title" and "url" are ignored	#}
+		{% set cols = [{text: rowData.title}, rowData] %}
+
+		{% set linkCol =  {
```
Comment (BenSurgisonPINS) [inline apps/web/src/server/views/applications/case/components/case-details-table-legacy.component.njk:26]:
```suggestion
		{% if not isPublishPage %}
			{% if rowData.url %}
				{% set linkCol =  {
					html: caseEditLink({editStepUrl:rowData.url, caseId: params.case.id}),
					classes: 'text-align--right pins-table__cell--s'
				} %}
                        {% else %}
				{% set linkCol =  {} %}
                        {% endif %}
			{% set cols = (cols.push(linkCol), cols) %}
		{% endif %}
```

Comment (BenSurgisonPINS) [reply apps/web/src/server/views/applications/case/components/case-details-table-legacy.component.njk:26]: Also here

Comment (BenSurgisonPINS) [reply apps/web/src/server/views/applications/case/components/case-details-table-legacy.component.njk:26]: I think the link should not be added if it doesn't exist in the rowData

**Review by BenSurgisonPINS — CHANGES_REQUESTED** [review]
Looks good, just an issue with the links in the case-details-table components

**Review by BenSurgisonPINS — APPROVED** [review]
lgtm

**Review by BenSurgisonPINS — APPROVED** [review]
lgtm

**Pull request merged**